### PR TITLE
Makefile fixes

### PR DESCRIPTION
**Pull request by bigjonroberts**
@gtmanfred @WritingPanda 
This Makefile includes your suggested changes.
The dependencies are properly installed by `make` as far as I can tell.
No bueno :(

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,13 +61,19 @@ help:
 	@echo 'Set the RELATIVE variable to 1 to enable relative urls                    '
 	@echo '                                                                          '
 
-html:
+icalendar:
+	python -c "import icalendar" || pip install icalendar
```
Comment (WritingPanda) [inline Makefile]: This part is not necessary. Also, it is not necessary to install icalendar every time HTML is run.

Comment (WritingPanda) [reply Makefile]: What I mean by "this part," I am referring to the python -c "import icalendar" bit.

**Comment by bigjonroberts** [comment]
```
make publish
python -c "import icalendar" || pip install icalendar
pelican /c/work/sadevs/sadevs.github.io/content -o /c/work/sadevs/sadevs.github.io/output -s /c/work/sadevs/sadevs.github.io/publishconf.py
CRITICAL: TypeError: makedirs() got an unexpected keyword argument 'exist_ok'
Makefile:106: recipe for target 'publish' failed
make: *** [publish] Error 1
```

File: Makefile (hunk 1 above)
Comment (bigjonroberts) [reply Makefile]: it is not installing icalendar every time HTML is run.  learn you some Makefile before you comment.

Comment (bigjonroberts) [reply Makefile]: and it is necessary, unless you want to put icalendar as a dependency in the docs

Comment (bigjonroberts) [reply Makefile]: trust me - this code does exactly what it is intended to do.
We are still missing a dependency.  That's what I am asking you to resolve.

Comment (bigjonroberts) [reply Makefile]: commenting on code you haven't run and don't understand is not helpful.  It's just a waste of everyone's time.  Unless the comment is "does this do x?"

**Pull request merged**